validator/client/src/main/java/tech/pegasys/teku/validator/client/duties/BlockProductionDuty.java

```diff
@@ -125,7 +125,7 @@ private SafeFuture<BlockContainer> validateBlock(
 
     if (!blockContainerAndMetaData.consensusBlockValue().isZero()) {
       LOG.info(
-          "Received block for slot {}, block rewards {} ETH, execution payload value {} ETH",
+          "Validator client received block for slot {}, block rewards {} ETH, execution payload value {} ETH",
```

Comment:
This is annoying me for a while.
Reading the logs is unclear who is saying this. Better to clarify that is VC speaking here.